### DISCUSSION TITLE
disable automatic translation?

### DISCUSSION THREAD

**Discussion by vinsse2001** [openai/whisper#1894] [Q&A]
Hello everyone!
is it possible to generate a transcript in 2 languages, keeping them without translation? for example, a speaker gives a talk in english, then is translated by someone in french, continues in english, is translated, etc.? The result is an audio file alternating between the 2 languages.
Is it possible to generate the transcript without automatically translating the English, and thus keep the 2 languages?
Or could the transcript be generated for just one of the 2 languages?
Thank you very much!

**Comment by sproocht** [comment]
Hi @vinsse2001,
The model is multilingual and is able to switch between different languages, especially high resource ones like English, French ... . Now, the transcription will be even more accurate when you use a fine-tuned model for French in your case.
All the best!

**Reply by vinsse2001** [reply]
Hi @sproocht,
I already use the `--language French` parameter, which gives me a perfectly correct transcription. The translation seems to work fine too, but that's not the problem.
The problem is to have an automatic translation, so the transcript isn't very readable: everything is in French, with no differentiation between the 2 speakers, so it's harder to find your way around the transcript. And as I use these transcripts to study them afterwards, I prefer to keep the transcript in English AND in French successively (or possibly, only have the transcript of the French part of the audio).
I'm surprised there isn't an option to deactivate the translation, so that I can use either the transcript, or the translation, or both?

**Reply by mattwong05** [reply]
Still having the same issue in 2026. Really wish there were an option to disable auto-translation.

**Reply by vinsse2001** [reply]
I use https://turboscribe.ai/ (based on whisper), it is very good :)

**Reply by mattwong05** [reply]
> I use https://turboscribe.ai/ (based on whisper), it is very good :)

Thanks for the recommendation! TurboScribe does look really good :)

But in my case, the whole pipeline has to run locally for privacy/offline reasons, so cloud-based solutions unfortunately won’t work for me.
